numbers_letters_and_variables/number_of_seconds.rb

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env ruby
+
+seconds_in_min = 60
+min_in_hour = 60
+hours_in_day = 24
+days_in_week = 7
+interger_valid = false
+
+#year =
+#
+#puts "what's your current age?"
+#
+#while interger_valid != true
+#begin
+#  current_age = gets.chomp
+#  n = Integer(current_age)
+#  puts n
+#rescue ArgumentError
+#  puts"not an interger! please try again."
+#  interger_valid = false
+#else
+#  interger_valid = true
+#end
+#end
+
+puts "There are #{seconds_in_min} seconds in a minute"
+puts "There are #{min_in_hour} minutes in an hour"
+puts "There are #{hours_in_day} hours in a day"
+puts "There are #{days_in_week} days in a week"
+puts "That means there are:"
+puts "  #{seconds_in_min * min_in_hour} seconds in an hour,"
+puts "  #{seconds_in_min * min_in_hour * hours_in_day} seconds in an day,"
+puts "  #{seconds_in_min * min_in_hour * hours_in_day * days_in_week} seconds in a week"
```

Comment:
IMO, too much being done in interpolation.  Pull this computation out into a variable?